tools/testbench/topology_ipc4.c

```diff
@@ -1358,7 +1358,7 @@ static int tb_kcontrol_cb_new(struct snd_soc_tplg_ctl_hdr *tplg_ctl,
 				tplg_bytes->priv.size);
 			return -EINVAL;
 		}
-		memcpy(ctl->data, tplg_bytes->priv.data, tplg_bytes->priv.size);
+		ctl->data = tplg_bytes->priv.data;
```

Comment:
Just double-checking this tplg_bytes persists the same time period as 'ctl' so this is safe to do?